Create lobster-bisque.md (#433)
LukeSmithxyz/based.cooking: src/lobster-bisque.md

3. Cut it down the middle and remove the meat from the shell, set aside the meat for later.
4. Place a stockpot on medium high heat and heat up enough oil to cover the bottom of the pan, add the lobster shells and cook them until they are bright red.
5. Remove the oil from the pan and discard it, add your butter to the shells.
6. Add all diced vegetables and let it sauté for 2 minutes, add a dash of cognac and flambé, lower the heat to medium low.
7. Rougly dice the tomatoes and add them together with the tomato puree, once again let it sauté for 2 minutes.
8. Add the flour and give it all a stir, let it dry up in the pan.
9. Deglaze the pan with the white wine and once the wine has dried up a bit add the fish stock.
10. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bitesized pieces.
11. After the soup is done boiling run it through a fine sieve.
12. Bring the strained soup back to  a light boil and add the diced lobster pieces.
13. Let the soup boil for 4 more minutes to heat up the lobster and serve, finish of in the plate with a bite of heavy cream.

## Contribution

Yiusa

> -eth '0x68f1317c6512f0267fa711cafb6c134ae968fa80'

Reviewer: Not sure if tagging your ETH wallet is allowed.

Author: I've tagged it on all my recipes i have posted here and they all got added before, if monero and btc are allowed i don't see why eth wouldn't be.

Reviewer: Ok. Yeah I didn't read your other PRs closely

Reviewer 2: BTC and XMR addresses are allowed, what's so different about ETH?

Reviewer 3: Eth is obviously fine, but the format should be (in an unordered list):

```
- Yiusa, eth `0x68f1317c6512f0267fa711cafb6c134ae968fa80`
```

I've been going around and fixing yours to make them consistent with others.

# Lobster bisque

Creamy french lobster soup perfect for the hollidays, you can optionally replace the lobster with any other shellfish.

- ⏲️ Prep time: 20
- 🍳Cook time: 30 min
- 🍽️ Servings: 2 litres

## Ingredients

- 1 live lobster that weighs ~750 grams
- 2 onions
- 2 carrots
- 1 shallot
- 2 garlic cloves
- cognac
- 3 tbsp concentrated tomato puree
- 4 tomatoes 
- 3 tbsp flour
- 1 glass dry white wine
- 2 litres fish stock
- 1 bayleaf
- 2 twigs of thyme
- fresh parsley
- heavy cream
- canola oil
- 2 tbsp butter

## Directions

1. Finely dice the onions, carrots, shallots and garlic and set aside for later.
2. Blanche the lobster in boiling water for 2 minutes, remove from the water and let it leak.

> 3. Cut it down the middle and remove the meat from the shell, set aside the meat for later.

Reviewer 4: ```suggestion
3. Cut it down the middle and remove the meat from the shell, set the meat aside for later.
```

4. Place a stockpot on medium high heat and heat up enough oil to cover the bottom of the pan, add the lobster shells and cook them until they are bright red.
5. Remove the oil from the pan and discard it, add your butter to the shells.
6. Add all diced vegetables and let it sauté for 2 minutes, add a dash of cognac and flambé, lower the heat to medium low.
7. Rougly dice the tomatoes and add them together with the tomato puree, once again let it sauté for 2 minutes.
8. Add the flour and give it all a stir, let it dry up in the pan.
9. Deglaze the pan with the white wine and once the wine has dried up a bit add the fish stock.
10. Tie up your herbs using kitchen twine and add them to your pot.

> 11. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bitesized pieces.

Reviewer 4: ```suggestion
11. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bite-sized pieces.
```

12. After the soup is done boiling run it through a fine sieve.

> 13. Bring the strained soup back to  a light boil and add the diced lobster pieces.

Reviewer 4: ```suggestion
13. Bring the strained soup back to a light boil and add the diced lobster pieces.
```

14. Let the soup boil for 4 more minutes to heat up the lobster and serve, add pepper and salt to taste.

> 15. Finish of in the plate with a bite of heavy cream.

Reviewer 4: ```suggestion
15. Finish of in the plate with a bit of heavy cream.
```

Author: Sadly i don't, its a recipe i don't cook often. I could look online for an image but it won't exactly match the recipe. Also im not 100% sure on how to add an image to a recipe in the first place. New to git, just like to cook and hate modern recipe websites.

Reviewer 4: There is a rule against using images from google, so I wouldn't do that. All images here are taken by the creator or someone that used the recipe to make it.

For future, all you do is add an image file to the `data/pix` folder. Use the same name as the recipe file. If you have additional images for different steps of the recipe, then they will have the same name but will have numbers. example:

```
data/pix/lobster-bisque.webp
data/pix/lobster-bisque-01.webp
```

To add the image to the `.md` file you just add something like this, just look at how they are referenced in other recipe files.

Also, all images should be `webp` files, and less than 150kB

(A lot of this info is also on the main readme btw)

> 15. Finish of in the plate with a bite of heavy cream.

Reviewer 4: ```suggestion
15. Finish off in the plate with a bite of heavy cream.
```
